api/tests/routes/backoffice_v3/offerers_test.py

```diff
@@ -92,9 +89,70 @@ def test_get_offerer(self, authenticated_client):  # type: ignore
         assert offerer.name in content
         assert str(offerer.id) in content
         assert offerer.siren in content
+        assert offerer.city in content
+        assert offerer.postalCode in content
+        assert offerer.address in content
         assert "Structure" in content
 
 
+class UpdateOffererTest:
+    class UnauthorizedTest(unauthorized_helpers.UnauthorizedHelperWithCsrf):
+        method = "post"
+        endpoint = "backoffice_v3_web.offerer.update"
+        endpoint_kwargs = {"offerer_id": 1}
+        needed_permission = perm_models.Permissions.MANAGE_PRO_ENTITY
+
+    def test_update_offerer(self, client, legit_user):
+        offerer_to_edit = offerers_factories.OffererFactory()
+
+        old_city = offerer_to_edit.city
+        new_city = "Brest"
+        old_postal_code = offerer_to_edit.postalCode
+        new_postal_code = "29000"
+        expected_new_region = "Bretagne"
+        old_address = offerer_to_edit.address
+        new_address = "1 Rue de Siam"
+
+        base_form = {
+            "city": new_city,
+            "postal_code": new_postal_code,
+            "address": new_address,
+        }
+
+        response = self.update_offerer(client, legit_user, offerer_to_edit, base_form)
+        assert response.status_code == 303
+
+        expected_url = url_for("backoffice_v3_web.offerer.get", offerer_id=offerer_to_edit.id, _external=True)
+        assert response.location == expected_url
+
+        client.with_bo_session_auth(legit_user).get(expected_url)
+        history_url = url_for("backoffice_v3_web.offerer.get_details", offerer_id=offerer_to_edit.id)
+        response = client.with_bo_session_auth(legit_user).get(history_url)
+
+        offerer_to_edit = offerers_models.Offerer.query.get(offerer_to_edit.id)
+        assert offerer_to_edit.city == new_city
+        assert offerer_to_edit.postalCode == new_postal_code
+        assert offerer_to_edit.address == new_address
+        assert regions_utils.get_region_name_from_postal_code(offerer_to_edit.postalCode) == expected_new_region
```

Comment:
Cette ligne ne sert à rien ici, le but n'est pas de tester `get_region_name_from_postal_code`, et on a déjà vérifié le code postal au-dessus.
Tu peux tester `assert expected_new_region in content` plus bas.